src/gluonts/model/deepvar_hierarchical/_network.py

```diff
@@ -32,6 +33,106 @@
 )
 
 
+def reconcile_samples(
```

Comment:
This function is just moved out of the network and `if` and `else` blocks are swapped for better readability.